sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+
+import { ICustomHeaders, IOutputVariables } from "zeebe-node";
+import {
+  createFakeWorkerJob,
+  FAKE_WORKER_JOB_RESULT_PROPERTY,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../../test/helpers";
+import { createFakeWorkflowWrapUpPayload } from "./workflow-wrap-up";
+import { AssessmentController } from "../../assessment.controller";
+import { WorkflowWrapUpJobInDTO } from "../../assessment.dto";
+import {
+  FormYesNoOptions,
+  RelationshipStatus,
+  StudentAssessmentStatus,
+  WorkflowData,
+} from "@sims/sims-db";
+
+describe("AssessmentController(e2e)-workflowWrapUp", () => {
+  let db: E2EDataSources;
+  let assessmentController: AssessmentController;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    db = createE2EDataSources(dataSource);
+    assessmentController = nestApplication.get(AssessmentController);
+  });
+
+  it(`Should update assessment status to completed when it does not have a status ${StudentAssessmentStatus.CancellationRequested}, ${StudentAssessmentStatus.CancellationQueued} or ${StudentAssessmentStatus.Cancelled}.`, async () => {
+    // Arrange
+    const savedApplication = await saveFakeApplication(db.dataSource);
+    const workflowData: WorkflowData = {
```

Comment:
Is there a reason to have an extended `workflowData` when testing its content is not the actual goal of the tests?
It seems to me that the other E2E tests in this PR are not following the same and they are using a more simple payload for a similar scenario. Please let me know if I am missing something.

Reply:
Changed to a simpler workflowData. Thanks!

Reply:
Minor but we probably do not need the explicit type here `workflowData: WorkflowData` and the `as WorkflowData` also.